### ISSUE TITLE
[🐞 BUG]: Sidebar is different!

### ISSUE THREAD

**Issue by ImBIOS** [vercel-labs/ai-chatbot#175]
The cloned version's (https://github.com/vercel-labs/ai-chatbot/) and the production version ([chat.vercel.ai](https://chat.vercel.ai/))'s sidebar is different

![image](https://github.com/vercel-labs/ai-chatbot/assets/41441643/cc3e2da9-6d53-4e85-83a0-900fc0235d91)

**Comment by Saran33** [comment]
There seems to be numerous differences between the demo site and this repo. They seem to have diverged since Autumn '23. It's awesome and really grateful for this amazing project, but at the same time it's somewhat disappointing misleading to use a demo site that differs from the actual open source code.

**Comment by GabrielDucharme** [comment]
Yes it's pretty frustrating

**Comment by ImBIOS** [comment]
I successfully imitated the demo website this past weekend, want to see it? I do it just for fun.

**Comment by dariogcode** [comment]
> I successfully imitated the demo website this past weekend, want to see it? I do it just for fun.

Yes please, can you share how to do you revert to that version. I've inpsected git log but didn't found a proper commit to revert changes in sidebar.

**Comment by spookymargiela007** [comment]
@ImBIOS it'd be awesome if you can share you fix!

**Comment by ImBIOS** [comment]
You're welcome, https://github.com/ImBIOS/ai-chatbot-t3
(Sorry, I can't spend time to remove the t3 stack, I'm currently experimenting with this 'type-safe cult' thing)

I'm working on the share button beside the regenerate response button, but I'm not yet done. If someone wants to open a PR, welcome.

**Comment by 0x-Legend** [comment]
> I successfully imitated the demo website this past weekend, want to see it? I do it just for fun.

yes please!

**Comment by leerob** [comment]
Apologies that the demo is slightly different, it's just a styling change. We are thinking about redesigning that bit but for now it shouldn't impede on using this template to get started 🙏
